sql: single-column EXPLAIN output

RaduBerinde opened the pull request:
#### sql: single-column EXPLAIN output

This change replaces the EXPLAIN columns with a single string column
and shows the information in a tree.

Release note (sql change): New single-column output format for EXPLAIN
and EXPLAIN (VERBOSE).

#### sql: update tests involving EXPLAIN

Release note: None

RaduBerinde committed:
sql: update tests involving EXPLAIN

Release note: None

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.png" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/55866)
<!-- Reviewable:end -->

asubiotto reviewed (approved):
<img class="emoji" title=":lgtm:" alt=":lgtm:" align="absmiddle" src="https://reviewable.io/lgtm.png" height="20" width="61"/> 

> Reviewed 6 of 6 files at r1, 80 of 80 files at r2.
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/55866#-:-MKZEnl_09vfhqee-96w:bg7igkv)* status: :shipit: complete! 1 of 0 LGTMs obtained (waiting on @jordanlewis)



<!-- Sent from Reviewable.io -->

RaduBerinde committed:
sql: single-column EXPLAIN output

This change replaces the EXPLAIN columns with a single string column
and shows the information in a tree.

Release note (sql change): New single-column output format for EXPLAIN
and EXPLAIN (VERBOSE).

RaduBerinde commented:
bors r+
